lib/identity_doc_auth/acuant/responses/get_results_response.rb

```diff
@@ -62,62 +57,28 @@ def pii_from_doc
 
         attr_reader :http_response
 
-        def generate_errors
-          return {} if successful_result?
-
-          dpi_threshold = config.dpi_threshold&.to_i || 290
-          sharpness_threshold = config&.sharpness_threshold&.to_i || 40
-          glare_threshold = config&.glare_threshold&.to_i || 40
-
-          front_dpi_fail, back_dpi_fail = false
-          front_sharp_fail, back_sharp_fail = false
-          front_glare_fail, back_glare_fail = false
-
-          processed_image_metrics.each do |side, img_metrics|
-            hdpi = img_metrics['HorizontalResolution'] || 0
-            vdpi = img_metrics['VerticalResolution'] || 0
-            if hdpi < dpi_threshold || vdpi < dpi_threshold
-              front_dpi_fail = true if side == :front
-              back_dpi_fail = true if side == :back
-            end
-
-            sharpness = img_metrics['SharpnessMetric']
-            if sharpness.present? && sharpness < sharpness_threshold
-              front_sharp_fail = true if side == :front
-              back_sharp_fail = true if side == :back
-            end
-
-            glare = img_metrics['GlareMetric']
-            if glare.present? && glare < glare_threshold
-              front_glare_fail = true if side == :front
-              back_glare_fail = true if side == :back
-            end
-          end
-
-          return { results: [Errors::DPI_LOW_BOTH_SIDES] } if front_dpi_fail && back_dpi_fail
-          return { results: [Errors::DPI_LOW_ONE_SIDE] } if front_dpi_fail || back_dpi_fail
-
-          return { results: [Errors::SHARP_LOW_BOTH_SIDES] } if front_sharp_fail && back_sharp_fail
-          return { results: [Errors::SHARP_LOW_ONE_SIDE] } if front_sharp_fail || back_sharp_fail
+        def response_info
+          @response_info ||= create_response_info
+        end
 
-          return { results: [Errors::GLARE_LOW_BOTH_SIDES] } if front_glare_fail && back_glare_fail
-          return { results: [Errors::GLARE_LOW_ONE_SIDE] } if front_glare_fail || back_glare_fail
+        def create_response_info
+          alerts = processed_alerts
 
-          #if no image metric errors default to raw error output
-          error_messages_from_alerts
+          {
```

Comment:
Should there be a `liveness_enabled` property in this hash?

Reply:
For TrueID we needed it for the work we were doing so it was obvious  to add it in there. I looked at adding it and had a placeholder there but it would just be for logging purposes for Acuant. The liveness handling is quite different due to the differences between requests for the two vendors. I'm hoping we can just throw out the Acuant code in July after TrueID has been on for a few weeks.